test/unit/notify/send-emails.test.js

```diff
@@ -28,25 +28,58 @@ describe('send emails', () => {
     defaultEmailAddresses = env.notifyEmailAddresses.split(',')
```

Comment:
Make sure your `standardjs` is configured and running in this repo -- mine flagged there's a trailing white line in this file